test/Npgsql.Tests/CommandTests.cs

```diff
@@ -447,6 +447,20 @@ public void CloseConnectionWithException()
 
         #endregion
 
+        [Test]
+        public void DuplicateViolation()
+        {
+    using (var conn = OpenConnection())
+    {
+        using (var cmd = new NpgsqlCommand("CREATE TABLE foo (id INT PRIMARY KEY)", conn))
+            cmd.ExecuteNonQuery();
+        using (var cmd = new NpgsqlCommand("INSERT INTO foo (id) VALUES (1)", conn))
+            cmd.ExecuteNonQuery();
+        using (var cmd = new NpgsqlCommand("INSERT INTO foo (id) VALUES (1)", conn))
+            cmd.ExecuteNonQuery();
+    }
```

Comment:
nit: Indentation.

Reply:
Was not supposed to be committed at all, thanks for catching.